src/cl/lodestar/lodestar_launcher.star

```diff
@@ -226,26 +225,18 @@ def get_beacon_config(
             el_client_context.engine_rpc_port_num,
         )
 
-    genesis_config_filepath = shared_utils.path_join(
-        GENESIS_DATA_MOUNT_DIRPATH_ON_SERVICE_CONTAINER,
-        genesis_data.config_yml_rel_filepath,
-    )
-    genesis_ssz_filepath = shared_utils.path_join(
-        GENESIS_DATA_MOUNT_DIRPATH_ON_SERVICE_CONTAINER,
-        genesis_data.genesis_ssz_rel_filepath,
-    )
-    jwt_secret_filepath = shared_utils.path_join(
-        GENESIS_DATA_MOUNT_DIRPATH_ON_SERVICE_CONTAINER,
-        genesis_data.jwt_secret_rel_filepath,
-    )
     cmd = [
         "beacon",
         "--logLevel=" + log_level,
         "--port={0}".format(DISCOVERY_PORT_NUM),
         "--discoveryPort={0}".format(DISCOVERY_PORT_NUM),
         "--dataDir=" + CONSENSUS_DATA_DIRPATH_ON_SERVICE_CONTAINER,
-        "--paramsFile=" + genesis_config_filepath,
-        "--genesisStateFile=" + genesis_ssz_filepath,
+        "--paramsFile="
+        + package_io.GENESIS_CONFIG_MOUNT_PATH_ON_CONTAINER
```

Comment:
can we bake this into a static constant as well????

Reply:
what exactly do you wanna bake in to static constant?

Reply:
https://github.com/kurtosis-tech/ethereum-package/pull/288/files/faf8e8e87ff4c31a3daaa176d647eb9b86e9335c#diff-660dcd34b28ff6d96e33a5ded2e696ba64b1e0012027d4603c0e98d043ae4575R236

the `config.yml` - https://github.com/kurtosis-tech/ethereum-package/pull/288/files/faf8e8e87ff4c31a3daaa176d647eb9b86e9335c#diff-660dcd34b28ff6d96e33a5ded2e696ba64b1e0012027d4603c0e98d043ae4575R236 

`CONFIG_YAML_PATH_ON_CONTIANER` ?

Reply:
I'd be happy to take those out to a global if all CLs would take config.yaml file as their input, but LH for example needs the whole network directory not the config file, and would break if you pass the config to it. Its too custom to make it a global, or we would need to have more globals, which kinda defeats the purpose.